Zoom.cpp

```diff
@@ -223,10 +226,13 @@ void VertZoomRange(int iFirst, int iNum, bool* bZoomed, bool bMinimizeOthers, bo
 			if (bZoomed[i])
 			{
 				if (i + 1 == iNum)
-					iEachHeight +=leftOverHeight;
+					iEachHeight += leftOverHeight;
 				MediaTrack* tr = CSurf_TrackFromID(i + iFirst, false);
 				if (!obeyHeightLock || !GetMediaTrackInfo_Value(tr, "B_HEIGHTLOCK"))
+				{
 					GetSetMediaTrackInfo(tr, "I_HEIGHTOVERRIDE", &iEachHeight);
+					iEachHeight += GetTrackSpacerSize(tr);
```

Comment:
Each selected track should end up with the same height, this line increases it once per track: https://i.imgur.com/tdFgLGS.png

Pushing a fix. Rebased off master to test with the workaround for "minimize others" being completely broken in v6.76+.

Also renamed `GetTrackHeightAndOffsetWithSpacers` to `GetTrackHeightWithSpacer` because `GetTrackHeight` anonymously also does the `AndOffset` part (and this was getting long to type).

Reply:
Thanks, sorry about that!